Formula/httpie.rb

```diff
@@ -3,8 +3,8 @@ class Httpie < Formula
 
   desc "User-friendly cURL replacement (command-line HTTP client)"
   homepage "https://httpie.io/"
-  url "https://files.pythonhosted.org/packages/e9/38/e94dac67b61f4dab49c1d26dd47e0b13be8c69c8c1c4fad5a4a87de1d647/httpie-3.2.1.tar.gz"
```

Comment:
maybe a pypi artifact issue?

Reply:
I think this breaks `update-python-resources`, so we'll want to work out a way to avoid that.

Reply:
> I think this breaks `update-python-resources`, so we'll want to work out a way to avoid that.

Ah, I see. We could just download the two files as additional resources instead.

Reply:
> Ah, I see. We could just download the two files as additional resources instead.

I think `update-python-resources` also has issues with non-PyPI resources.

Could try `pypi_formula_mappings.json` and add a line like `"httpie": "httpie",` and try locally testing if `brew update-python-resources httpie` still works.